### ISSUE TITLE
GitWriter should only list user-owned repos in the "[user] repositories" tab

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-DevGitWriter#2]
## Expected Behaviour

For GitHub user ilovan, only these repositories should be listed:
https://github.com/ilovan?tab=repositories

## Current Behaviour

The tab currently lists all the user's repos, but also all the repos belonging to a GitHub organization the user is a member of:

![image.png](https://images.zenhubusercontent.com/5a5e621f4b5806bc2bc2ff7d/4e88f275-405c-4ea1-a354-f85b2b2fd4d6)

## Discussion:
@SusanBrown , could you please weigh in this one?
I believe that including all the repos of an organization the user belongs to into the "[user] repositories" raises some questions of ownership and accountability that may better be left out.

**Comment by ilovan** [comment]
lists also the repos the current user is a collaborator on (e.g. in the screenshot - Connie's LGLC repo)

**Comment by jchartrand** [comment]
I could look at adding a dropdown or radio buttons with options like:  ‘show only my repositories’, ‘show repositories for all organizations to which I belong'

**Comment by SusanBrown** [comment]
I agree that this is too broad as currently worded.

What is the estimate on providing a radio button as opposed to rewording the tab? This would be the preferred solution.

Could be reworded "Repositories 'ilovan' can edit" [if that is in fact the case, but if membership does not necessarily mean that, then "repositories with member 'ilovan'"]

Suggest rewording "Search for documents" rather than "Search my documents" regardless of which route we take.

**Comment by jchartrand** [comment]
Adding the radio buttons is easy.  Making them work with the GitHub api depends on the Github API itself, but I can check it out tomorrow, and give a better estimate then, if it's worth checking out.  Rough guess though:  4 to 8 hours?

**Comment by jchartrand** [comment]
I looked into it this morning, and it should be fine to add filters (probably checkboxes, but could be a multiselect dropdown) to show only repositories:

- that are owned by the user
- for which the user is a collaborator
- that are owned by organizations of which the user is a member

or any combination of the three.

A day's work is still a good guess.
